quantum/stm32/chconf.h

```diff
@@ -58,7 +58,7 @@
  *          The value one is not valid, timeouts are rounded up to
  *          this value.
  */
-#define CH_CFG_ST_TIMEDELTA                 2
+#define CH_CFG_ST_TIMEDELTA                 0
```

Comment:
This may (does not) need to be changed if `STM32_PWM_USE_TIM2` is set to false;

Or if `STM32_ST_USE_TIMER` was changed to something other than 2.